functional-tests/build.gradle

```diff
@@ -2,175 +2,155 @@ import org.apache.tools.ant.taskdefs.condition.Os
 import org.gradle.api.tasks.testing.logging.TestExceptionFormat
 import org.gradle.api.tasks.testing.logging.TestLogEvent
 
-// Shelly's note: setup baseURL env var to take test/dev
-
-
 plugins {
-    id 'com.gradle.build-scan' version '1.16'
-    id "groovy"
-    id "com.energizedwork.webdriver-binaries" version "1.0"
-    id "net.foragerr.jmeter" version "1.0.10-3.3"
-    
+  id 'com.gradle.build-scan' version '1.16'
```

Comment:
is there a reason why this one is wrapped in single quotes and the rest in doubles?